assets/js/theme/common/carousel/utils/tooltipSetup.js

```diff
@@ -1,23 +1,21 @@
-const carouselTooltipClass = 'carousel-tooltip';
-const carouselTooltip = `<span class="${carouselTooltipClass}"></span>`;
+export const setupAriaAttributes = ($node, ariaAttribute, attributeValue, ...accessibiltyClassList) => {
```

Comment:
can we also separate in a file as it is new util?

Reply:
done